src/platform-integration/web/web-images.md

```diff
@@ -1,37 +1,41 @@
 ---
-title: Displaying images on the web
+title: Display images on the web
 short-title: Web images
 description: Learn how to load and display images on the web.
 ---
 
-The web supports the standard [`Image`][1] widget to display images.
+The web supports the standard [`Image`][] widget to display images.
 However, because web browsers are built to run untrusted code safely,
 there are certain limitations in what you can do with images compared
 to mobile and desktop platforms. This page explains these limitations
 and offers ways to work around them.
 
-# Background
+[`Image`]: {{site.api}}/flutter/widgets/Image-class.html
+
+## Background
 
 This section summarizes the technologies available
 across Flutter and the web,
 on which the solutions below are based on.
 
-## Images in Flutter
+### Images in Flutter
 
-Flutter offers the [`Image`][1] widget as well as the low-level
-[`dart:ui/Image`][11] class for rendering images.
+Flutter offers the [`Image`] widget as well as the low-level
```

Comment:
Is this a valid link format in MD?

Reply:
Oops, thanks for calling that out.

It is, but we prefer to use the full `[text][]` format for consistency with `[text][link definition]` syntax.